requirements.txt

```diff
@@ -1,22 +1,6 @@
 # The python wrappers to libecl and libres are required by ert, but there is not
-# yet any pip package for those, they are therefor just mentioned here with a comment, 
+# yet any pip package for those, they are therefore just mentioned here with a comment,
```

Comment:
Should we not just delete this file?

Reply:
The reason I didn't is because it still details the installation of `libres` and `libecl`. You feel that should just be written in the `README`?

Reply:
Yes, I think that is better to explain in the readme then in a comment in the `requirements.txt`-file ;)